### PR TITLE
added mechanic of player movement

### PR DESCRIPTION
**Pull request by REFLAXua**
watch what i done

### CODE REVIEW

**Review by deadsnxw — COMMENTED** [review]
1. Unnecessary variable "timer"
2. Useless comment

File: lab4/Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    public float moveSpeed = 5f;
+    private Rigidbody2D rb;
+    public int timer;
```
Comment (deadsnxw) [inline lab4/Assets/Scripts/PlayerMovement.cs:7]: Unnecessary variable

Comment (deadsnxw) [inline lab4/Assets/Scripts/PlayerMovement.cs:7]: Unnecessary variable

File: lab4/Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    public float moveSpeed = 5f;
+    private Rigidbody2D rb;
+    public int timer;
+
+
+    //{}
```
Comment (deadsnxw) [inline lab4/Assets/Scripts/PlayerMovement.cs:10]: Useless comment

File: lab4/Assets/Scripts/PlayerMovement.cs (hunk 1 above)
Comment (REFLAXua) [reply lab4/Assets/Scripts/PlayerMovement.cs:7]: oh sure, forgot about it

**Review by REFLAXua — COMMENTED** [review]
must delete useless variable and comment

**Pull request closed without merging**